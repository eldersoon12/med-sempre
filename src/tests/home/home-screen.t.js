import HomeScreen from "../../pages/home";
import { renderComponentTest } from "..";

renderComponentTest("renders correctly", <HomeScreen />);
