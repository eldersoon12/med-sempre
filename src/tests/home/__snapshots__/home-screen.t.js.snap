// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      [
        {
          "fontFamily": "Inter",
          "fontSize": 48,
        },
        {
          "color": "rgba(103, 80, 164, 1)",
        },
      ]
    }
  >
    Open up App.js to start working on your app!
  </Text>
</View>
`;
